spec/services/charges/validators/percentage_service_spec.rb

```diff
@@ -79,7 +114,14 @@
         }
       end
 
-      it { expect(percentage_service.validate.error).to include(:invalid_free_units_per_events) }
+      it 'is invalid' do
+        aggregate_failures do
+          expect(percentage_service).not_to be_valid
+          expect(percentage_service.result.error).to be_a(BaseService::ValidationFailure)
+          expect(percentage_service.result.error.messages.keys).to include(:free_units_per_events)
+          expect(percentage_service.result.error.messages[:free_units_per_events]).to include('invalid_free_units_per_events')
```

Comment:
Layout/LineLength: Line is too long. [126/120] (https://rubystyle.guide#max-line-length)